src/components/Footer.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { FC } from 'react';
-import styled from '../styling/styled';
 import { Link } from '@reach/router';
+import { FaFacebook, FaGithub } from 'react-icons/all';
```

Comment:
This package is quite big and we use only few icons. I'm not sure if we can change it now, but I think that we should take care of this in the future - maybe it will be better to have our own icons?

Reply:
This should be tree-shakeable, we need to first check if the current production bundle really contains all icons. I found an old thread on this, so it might be already working out of the box: https://github.com/react-icons/react-icons/issues/154

Reply:
Considering their website it is tree-shakable but only when using particular icon sets not all icons. (eg. `import { FaFacebook, FaGithub } from 'react-icons/fa'`).
![image](https://user-images.githubusercontent.com/20800459/104792981-c667b000-57a0-11eb-84bd-519260f8c50d.png)